pkg/framework/objects/starburst_credential/schema.go

```diff
@@ -0,0 +1,87 @@
+package starburst_credential
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	datasource_schema "github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+var resourceSchema = resource_schema.Schema{
+	Description: "Starburst credential resource",
```

Comment:
My only comment would be that this can be used for Starburst but also for Trino credentials

Reply:
Good catch, I did intend to reference Trino everywhere.